packages/core/src/components/service-message/components/ExpansionToggle.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { Icon, EDSStyleSheet, useStyles } from "@equinor/mad-components"
+
+type ExpansionToggleProps = {isExpanded: boolean}
+export const ExpansionToggle = ({isExpanded}: ExpansionToggleProps) => {
+    const styles = useStyles(theme);
+    const icon = isExpanded ? "menu-up" : "menu-down";
+    return <Icon name={icon} style={styles.icon} size={24}/>
+}
+
+const theme = EDSStyleSheet.create((themeStyles) => ({
+    icon: {
+        color: themeStyles.colors.text.feedbackWarning,
+        marginRight: -16,
```

Comment:
What is the reason for this negative margin?

Reply:
I tried to correct for the padding from the container. However, the padding was incorrect. I switched the container padding to `dialog` instead, and removed this margin.